README.md

```diff
@@ -87,6 +100,15 @@ The `podSpec` of the `kubernetes` plugin can support any field from the `PodSpec
 
 More samples can be found in the [integration test fixtures directory](internal/integration/fixtures).
 
+### Buildkite Clusters
+If you are using [Buildkite Cluster](https://buildkite.com/docs/agent/clusters) to isolate sets of pipelines from each other, you will need to specify the cluster's UUID in the configuration for the controller. This may be done using a flag on the `helm` command like so: `--set config.cluster-uuid=<your cluster's UUID>`, or an entry in a values file.
+```yaml
+# values.yaml
+config:
+  cluster-uuid: beefcafe-abbe-baba-abba-deedcedecade
+```
+The cluster's UUID may be obtained from the settings page for the cluster.
```

Comment:
Perhaps we could offer a helpful link here:

https://buildkite.com/organizations/-/clusters

Reply:
The tricky thing about a link is the page I want to link to is:
`https://buildkite.com/organizations/-/clusters/:cluster-uuid/edit`, but you need to know the information you seek in order to enter it as a URL.

Maybe this might help though:
```suggestion
The cluster's UUID may be obtained by navigating to the [clusters page](https://buildkite.com/organizations/-/clusters), clicking on the relevant cluster and then clicking on "Settings".
```

Reply:
Yeah, nice